conductr_cli/resolver.py

```diff
@@ -64,13 +65,16 @@ def continuous_delivery_uri(custom_settings, resolved_version):
     raise ContinuousDeliveryError('Unable to form Continuous Delivery uri using {}'.format(resolved_version))
 
 
-def resolver_chain(custom_settings):
+def resolver_chain(custom_settings, offline_mode):
     log = logging.getLogger(__name__)
     if custom_settings is not None and 'resolvers' in custom_settings:
         resolver_names = custom_settings.get_list('resolvers')
         if resolver_names:
             log.info('Using custom bundle resolver chain {}'.format(resolver_names))
             custom_resolver_chain = [importlib.import_module(resolver_name) for resolver_name in resolver_names]
             return custom_resolver_chain
-
-    return DEFAULT_RESOLVERS
+    else:
+        if offline_mode:
+            return list(set(DEFAULT_RESOLVERS) - set(OFFLINE_RESOLVERS_EXCLUSION))
```

Comment:
Unsure why we must return a different list here. Each resolver should handle being offline (which I think they do). It feels leaky to assume which resolvers are for offline here.